frontend/src/locales/en.json

```
@@ -171,8 +175,19 @@
     },
     "print": {
       "config": {
+        "activityConfig": {
+          "activity": "Activity"
+        },
+        "picassoConfig": {
+          "orientation": "Page layout",
+          "periods": "Bearing section(s)"
```

🤣 Ja, das ist das Schöne an Übersetzungen ohne Kontext. Kugellager wollen wir hier keine (auch wenn es zum Kugeln ist), sondern:

```suggestion
          "periods": "Period(s)"
```
Unten auch noch 2x